guide/voice/voice-connections.md

```diff
@@ -59,6 +59,8 @@ if (subscription) {
 
 ::: warning
 **Voice connections can be subscribed to one audio player at most.** If you try to subscribe to another audio player while already subscribed to one, the current audio player is unsubscribed and replaced with the new one.
+
+It is also worth noting that the **GUILD_VOICE_STATES** [gateway intent](https://discordjs.guide/popular-topics/intents.html#gateway-intents) is required. Without it, the connection will permanently stay in the `signalling` state, and you'll be unable to play audio.
```

Comment:
```suggestion
It is also worth noting that the **GUILD_VOICE_STATES** [gateway intent](/popular-topics/intents.md#gateway-intents) is required. Without it, the connection will permanently stay in the `signalling` state, and you'll be unable to play audio.
```
It's better to use relative links instead of absolute ones.